.github/workflows/pypi.yml

```diff
@@ -0,0 +1,72 @@
+---
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+on:
+  # Only deploy if we're the result of a PR being merged
+  push:
+    branches:
+      - master
```

Comment:
Do you think that maybe it is better to use a label checker here? like this one? https://github.com/marketplace/actions/label-checker-for-pull-requests

We can make only push if there is some change in the pyln modules?